integration/tests/cook/test_basic.py

```diff
@@ -1326,19 +1328,21 @@ def test_attribute_equals_hostname_constraint(self):
         # Based on the priority, canary should be scheduled ahead of big_job, but because
         # of the host-placement constraint they have to be scheduled on the same host.
         # The canary should be able to be scheduled, but big_job should never be scheduled.
-        big_job = util.minimal_job(group=group['uuid'],
+        big_jobs = [util.minimal_job(group=group['uuid'],
                                    priority=1,
                                    cpus=max_cpus)
-        uuids, resp = util.submit_jobs(self.cook_url, [canary, big_job], groups=[group])
+                    for i in range(num_big_jobs)]
```

Comment:
You're not actually using the `i`, right?
```python
for _ in range(num_big_jobs)]
```

Reply:
Done